apps/concurrent/test/mns.plt

```diff
@@ -0,0 +1,55 @@
+:- begin_tests(mns).
+
+:- use_module("../modules/user_input_parsing.pl").
+:- use_module("../modules/query_parsing.pl").
+:- use_module("../concurrent.pl").
+:- use_module("../modules/mns.pl").
+
+% currently fails
+test('mns empty') :-
+    parse_domain("A causes f", Domain),
+    parse_obs("f|0", Obs),
+    mns([], 0, Domain, Obs, Some_Valid_MNS),
```

Comment:
Obs is not an assoc list. First parameter represents a set of all fluents that you want to execute. The fourth parameter should be an assoc list [fluent->its value] not Obs, Obs is **not used** in the MNS algorithm